kubejs/server_scripts/farmerscompat.js

```diff
@@ -2,32 +2,26 @@ onEvent("recipes", (event) => {
     mixingRecipeGen(event);
 })
 
-var FULL_BUCKET_AMMOUNT = 81000;
-var INGOT_FLUID_AMMOUNT = 9000;
+const FULL_BUCKET_AMOUNT = 81000;
+const INGOT_FLUID_AMOUNT = 9000;
 
 function mixingRecipeGen(event) {
     event.forEachRecipe({type:"farmersdelight:cooking"}, recipe => {
         //Gets item name without namespace (e.g. "cooked_rice")
-        var outputItem = recipe.getOriginalRecipeResult().getItem().getId().substring(recipe.getOriginalRecipeResult().getItem().getId().indexOf(":")+1); 
+        var outputItem = recipe.getOriginalRecipeResult().getId().split(":")[1]; 

```

Comment:
These should also be let; in general, usage of `var` is obsolete in javascript and you should only use `let` and `const`